org.omegat.OmegaT.yml

```diff
@@ -0,0 +1,69 @@
+app-id: org.omegat.OmegaT
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+command: run.sh
+finish-args:
+  - --filesystem=xdg-documents
+  - --socket=x11
+  - --share=ipc
+
+cleanup:
+  - '*.a'
+  - '*cmake*'
+  - share/include
+  - share/cmake*
+  - share/autoconf
+  - share/automake*
+  - share/bash-competions
+  - share/doc
+  - share/man
+  - share/vala
+  - share/info
+  - usr/bin/x86_64-unknown*
```

Comment:
is there a reason this is in the cleanup section? I would think the executable binaries would need to stay in the flatpak.

Reply:
These are not the application's binaries but part of the buildchain (C, C++ and Vala compilers) which get pulled in during the build. They are not needed to run the app.

Reply:
I don't see any of these files in the built app even though I removed `cleanup` section locally.